examples/test_all_node_examples.sh

```
@@ -0,0 +1,39 @@
+echo "Test 2: node-lite (WASM sync)"
+echo "=========================================="
+cd node-lite
+pnpm start 2>&1 | head -30
+echo
+cd ..
+
+# Test node-worker-main
+echo "=========================================="
+echo "Test 3: node-worker-main (Worker + JS/WASM)"
+echo "=========================================="
+cd node-worker-main
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)